src/config/app.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {PathLike} from 'fs';
+import createConfig from './app';
+
+describe('config/app', () => {
+    describe('createConfig()', () => {
+        const phpIniFromConfigurationPath: PathLike = 'tests/php-ini-from-configuration';
+
+        it('should return valid config', () => {
+            expect(createConfig(
+                {
+                    codeChecks : true,
+                    docLinting : true,
+                },
+                `${phpIniFromConfigurationPath}/composer.json`,
```

Comment:
What are we testing here? 🤔

Would probably make sense to have these files readable inside the test?

Reply:
Yeah, this was more just a quick example test, but it makes sense to have distinct tests.